examples/vanilla-es6/README.md

```diff
@@ -0,0 +1,38 @@
+# JavaScript ES6 (or ES2015) TodoMVC Example
```

Comment:
Can you make sure this whole readme follows the [template](https://github.com/tastejs/todomvc-app-template/blob/master/app-readme.md) standard?